arcaflow_plugin_uperf/uperf_schema.py

```diff
@@ -493,6 +494,14 @@ class Profile:
             "description": "A list of groups in the profile.",
         }
     )
+    comm_port: int = field(
+        default=20000,
+        metadata={
+            "name": "comm_port",
+            "description": "The port to connect to on the server. "
+                           "Necessary for proper client-server communication.",
+        },
+    )
```

Comment:
Again, "port to connect to on the server" is rather generic in this context.  We should be much more specific:  this is the port on the host which is running the Uperf slave (or, perhaps the "Uperf server process").  The "necessary" sentence is kind of overblown -- this is the mechanism used by the Uperf micro-benchmark workload.

Ditto line 520.  (In fact, I would be sorely tempted to refactor these two pieces of code to use a common definition....)

Reply:
I re-worded this. I don't have them using the same documentation, because one is connecting to it, and the other is opening the port.